packages/components/src/Card/src/Card.module.css

```diff
@@ -0,0 +1,24 @@
+.hop-Card {
+    /* Internal Variables */
+    --border-radius: var(--hop-shape-rounded-md);
+
+    /* Main */
+    --hop-Card-main-border: var(--hop-space-10) solid var(--hop-neutral-border-weak);
+    --hop-Card-main-background-color: var(--hop-neutral-surface);
+
+    /* Second-level */
+    --hop-Card-second-level-border: var(--hop-space-10) solid var(--hop-neutral-surface-weakest);
+    --hop-Card-second-level-background-color: var(--hop-neutral-surface-weakest);
+
+    border-radius: var(--border-radius);
```

Comment:
border and background color should be set here. The internal variables should be set below here:
`.hop-Card--main {
    --background-color: var(--hop-Card-main-background-color);
    --border: var(--hop-Card-main-border);
}

.hop-Card--second-level {
    --background-color: var(--hop-Card-second-level-background-color);
    --border: var(--hop-Card-second-level-border);
}`